### ISSUE TITLE
Offet inside the vscode thunderclient tab

### ISSUE THREAD

**Issue by paulo-dc** [rangav/thunder-client-support#1075]
**Describe the bug**
When I try to edit the body of a request, or want to copy part of the response, there is an offset beetween the cursor and the stuff really modified or copied.

![thunderclient_vscode_offset_1](https://user-images.githubusercontent.com/31652387/227170625-60e4f472-0e4f-4fbe-94f9-923e3e715afd.gif)

![thunderclient_vscode_offset_2](https://user-images.githubusercontent.com/31652387/227170791-3739ced2-56f6-45c8-aa7f-53e397832808.gif)

**To Reproduce**
just edit a body request, or copy a word for example from the response.

**Expected behavior**
No offset.

**Platform:** 
 - OS: Windows 10 / WSL 2 / Debian bullseye
 - vscode version: 1.76.2
 - extension version: v2.5.1

**Your Team Size Using TC:**
at least 2 users. Surely more around.

**Comment by rangav** [comment]
Hi @paulo-dc did you change any font settings in vscode settings?

**Comment by rangav** [comment]
Also please try this settings value 
https://github.com/rangav/thunder-client-support/issues/888#issuecomment-1321970690

**Comment by paulo-dc** [comment]
Oh yes !!!!

I sincerly apologize. Before posting my issue, I tried to see if anyone did face the same and a bug was already openned. But obviously, it seems I didn't try enough :( . My bad.

And, indeed, my issue is a duplicated one, and I modified the fonts in my vscode. I added a font in the list (I don't even use...), and that was the root cause of the issue.

Falling back to the usual default value : 

`Consolas, 'Courier New', monospace`

solves the issue !

Many many thanks for your reactivity, and also for this great tool I really appreciate.
